lib/__tests__/standalone-fix.test.js

```diff
@@ -226,3 +226,96 @@ describe('writing fixes to files', () => {
 		expect(fileContent.startsWith('\uFEFF')).toBe(true);
 	});
 });
+
+it('one rule being disabled', async () => {
+	const code = `
+		a {
+			color: red;
+		}`;
+
+	const result = await standalone({
+		code,
+		config: {
+			rules: {
+				indentation: [
+					2,
+					{
+						disableFix: true,
+					},
+				],
+			},
+		},
+		fix: true,
+	});
+
+	expect(JSON.parse(result.output)[0].errored).toBe(true);
```

Comment:
Rather than check for errors, can we instead check if the output itself has been changed (like the tests at the beginning of the file), e.g.:

```js
const code = "  a { color: red; }";

const result = await standalone({
    code,
    config: {
        rules: {
            indentation: [
                2,
                {
                    disableFix: true,
                },
            ],
        },
    },
    fix: true,
});

expect(result.output).toBe(code);
```

And:

```js
const code = "  a { COLOR: red; }";

const result = await standalone({
    code,
    config: {
        rules: {
            indentation: [
                2,
                {
                    disableFix: true,
                },
            ],
            'property-case': ['lower'],
        },
    },
    fix: true,
});

expect(result.output).toBe("  a { color: red; }");
```

Reply:
When we use `disableFix`, what we expect is that if there is an error in the check, the output still contains the error message instead of the initial code, so that we can find it easily.

Reply:
Fair point.